Sources/KituraWebSocket/WebSocketConnection.swift

```diff
@@ -48,6 +48,9 @@ public class WebSocketConnection {
     
     private var active = true
     
+    private let timer: DispatchSourceTimer = DispatchSource.makeTimerSource()
```

Comment:
Doesn't this mean every connection creates a DispatchSource, even if we don't use it? Can we only create one (in `init()`) if we need one?